widgets/contact/js/contact.js

```diff
@@ -0,0 +1,61 @@
+var SiteOriginContactForm = {
```

Comment:
Ah, yeah, I'm going to add an .editorconfig file. Should help prevent this in future.